samples/contrib/aws-samples/ground_truth_pipeline_demo/mini-image-classification-pipeline.py

```diff
@@ -22,13 +22,13 @@ def ground_truth_test(region='us-west-2',
     ground_truth_train_job_name='mini-image-classification-demo-train',
     ground_truth_validation_job_name='mini-image-classification-demo-validation',
     ground_truth_label_attribute_name='category',
-    ground_truth_train_manifest_location='s3://your-bucket-name/gt-demo-images/ground-truth-demo/train.manifest',
-    ground_truth_validation_manifest_location='s3://your-bucket-name/gt-demo-images/ground-truth-demo/validation.manifest',
-    ground_truth_output_location='s3://your-bucket-name/gt-demo-images/ground-truth-demo/output',
+    ground_truth_train_manifest_location='s3://your-bucket-name/mini-image-classification/ground-truth-demo/train.manifest',
```

Comment:
Just like to confirm it's mini - small image?

Reply:
Just a very small set of images